conda/environment-test-3.10.yml

```diff
@@ -1,14 +1,22 @@
-name: cunumeric-test
+name: legate-core-test
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - python=3.10
 
   # build
-  - cutensor>=1.3.3
+  - git
```

Comment:
Is git really necessary here?

Reply:
Git is used during the build to clone dependencies from source so they can be built.